core/src/main/scala/org/apache/spark/api/r/RBackendHandler.scala

```diff
@@ -88,6 +88,19 @@ private[r] class RBackendHandler(server: RBackend)
     ctx.close()
   }
 
+  //get classPath for static object. This addresses SPARK-5185
```

Comment:
Style nit: Space after `//` - Also could you expand the comment to say `Looks up a class given a class name. This function first checks the current class loader and if a class is not found, it looks up the class in the context class loader.`